packages/template-typescript-minimal/config/default.js

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+module.exports = {
+    app: {
+        extensions: [
+            ['@salesforce/extension-commerce-bm-seo', {}],
```

Comment:
the order in which `extension-commerce-bm-seo` does not matter anymore for the server-side navigation to work